### DISCUSSION TITLE
Skype下载页面部分图标无法正常显示

### DISCUSSION THREAD

**Discussion by KenwayDai** [F9y4ng/GreasyFork-Scripts#289] [问答专区(Question / Answer)] [answered]
你使用的浏览器（名称&版本），脚本管理器（名称&版本），脚本版本；
 - Microsoft Edge 121.0.2248.1 (Dev Channel), Tampermonkey 4.19.0, 2023.11.07.1

你发现渲染问题的具体网站地址；
 - https://www.skype.com/en/get-skype/

出现渲染问题的网页截图；
![image](https://github.com/F9y4ng/GreasyFork-Scripts/assets/12065395/89f979c4-7d23-4fa9-af87-bc87a9d58f85)

如果该网站需要登录才能访问，请提供测试账号以供登录测试。
 - 不需要登录

**Comment by F9y4ng** [comment, marked as answer]
方法一： 在**需要渲染的网页元素**中找到 `:not([class*='icon'])`，把它修改为`:not([class*='icon' i])`，保存为全局数据。

方法二： 在**需要渲染的网页元素**中追加`:not(.skypeFontIcon)`，保存为站点独享。
